src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -290,6 +290,17 @@ public static JsonObjectBuilder json(Workflow wf){
 
         return bld;
     }
+    
+    public static JsonObjectBuilder json(Dataverse dv, Boolean mini) {
```

Comment:
You can use primitive here.

```suggestion
    public static JsonObjectBuilder json(Dataverse dv, boolean minimal) {
```